p2p/node/api.go

```diff
@@ -206,7 +206,7 @@ func (p *P2PNode) requestAndWait(peerID peer.ID, location common.Location, data
 
 // Request a data from the network for the specified slice
 func (p *P2PNode) Request(location common.Location, requestData interface{}, responseDataType interface{}) chan interface{} {
-	resultChan := make(chan interface{}, 1)
+	resultChan := make(chan interface{}, 10)
 	// If it is a hash, first check to see if it is contained in the caches
```

Comment:
we need this

Reply:
Definitely

Reply:
Actually it should be peerManager.C_peerCount

Reply:
resultChan := make(chan interface{}, peerManager.C_peerCount)

Reply:
On second thought, leave it as 10. We'll fix it later because not everything uses c_peerCount